iib/workers/tasks/build.py

```diff
@@ -482,6 +489,10 @@ def _skopeo_copy(source, destination, copy_all=False, exc_msg=None):
     ]
     if copy_all:
         args.append('--all')
+    if dest_token:
```

Comment:
Should this section also add a dummy `--dest-authfile`?

Reply:
The latest released version of skopeo doesn't have support for `--dest-authfile` :disappointed: